src/stores/tokens.js

```diff
@@ -8,10 +8,14 @@ export const useTokensStore = defineStore('tokens', () => {
   const rawListedTokens = ref(null)
   const allTokens = ref(null)
   const selectedTokenName = ref(null)
+  const allTokensCount = ref(null)
```

Comment:
Personally, I am not really a fan of using `all`. We have not used that for other entities. I think it should explain something else. Could it make sense without using `all` ?

Reply:
@janmichek I used `all` because it refers directly to the entities stored in allTokens ref that you have originally implemented 2 lines above. I assume you chose that prefix to distinguish allTokens related things (unfiltered) from filtered things like rawListedTokens.

Alternative naming that comes to my mind is unfilteredTokens* but I like allTokens* more. Do you have any nice suggestions how to improve the naming?

Reply:
Oh that was me 😅  I will try to come up with something...

Reply:
I went through the code again and remember that the inconsistent and annoying `all` prefix is used on purpose as the data comes from different sources. Here make sense. Thanks for refreshing my initial thinking. No need for change